### ISSUE TITLE
feature to keep the already rendered dom elements.

### ISSUE THREAD

**Issue by 123anvar** [angular/components#19234]
#### Bug, feature request, or proposal:
feature to keep the already rendered dom elements.

#### What is the expected behavior?
Do not remove the rendered elements on  scrolling down.

#### What is the current behavior?
Already rendered elements gets removed on scrolling DOWN and gets re-rendered on scrolling back UP.

#### What are the steps to reproduce?
I am using normal cdkVirtualFor for rendering a grid.

#### What is the use-case or motivation for changing an existing behavior?
The grid row and column properties gets jumbled on scrolling up and down.

eg: enabled columns gets disabled on scrolling back UP.

#### Versions of 
CDK : 7.3.7
Angular : 8.2.14
OS : Windows 10
TypeScript : 3.5.3
Browsers : Chrome

**Comment by jelbourn** [comment]
@mmalerba IIRC this can already be accomplished with the right configuration, right?

**Comment by mmalerba** [comment]
Not quite, there are settings to control how many template instances are cached (though they're still removed from the DOM) as well as settings to control how much buffer to render, but there's no setting to disable removing already rendered items. 

It would definitely be possible to make a scrolling strategy that does that, but currently users would have to build it themselves. It might be nice if we added a setting for it to our built-in strategies.

**Comment by lukaonik** [comment]
I think it's not virtual scroll behavior, it's infinite scroll behavior, you are adding things whenever you scroll and they never get removed. check this library, maybe it helps
https://github.com/orizens/ngx-infinite-scroll

**Comment by 123anvar** [comment]
unfortunately I was pushed to use ngx-infinite-scroll and now I am using infinite scroll but preffered to use cdk-virtualfor instead. Okay will wait and see !

**Comment by crisbeto** [comment]
We are closing this issue, because we've determined, using a semi-automated process, that it is no longer relevant, has low engagement, or it does not have enough information to proceed. Feel free to comment if you think the issue should be reopened, or open a new issue with the latest version of our code instead.
